### DISCUSSION TITLE
Additional Database thought: thesession.org

### DISCUSSION THREAD

**Discussion by jacobdgm** [DDMAL/linkedmusic-datalake#6] [Q&A]
A thought: might we want to include https://thesession.org in the datalake? They have an enormous number of transcriptions of traditional dance tunes (mostly Irish, but also from a bunch of other traditions) in ABC notation (see https://thesession.org/tunes), as well as linked-ish data on recordings/albums (see https://thesession.org/recordings), performers (e.g. https://thesession.org/recordings/artists/2130) and composers (e.g. https://thesession.org/tunes/composers/128).

There also exist a bunch of other collections of ABC transcriptions around the web (https://www.montrealsession.ca comes to mind)

**Comment by dchiller** [comment]
This is v cool!

**Comment by fujinaga** [comment]
Can we get a database dump of this site, an API?

**Reply by malajvan** [reply]
They do have a dump of CSVs and JSONs (https://github.com/adactio/TheSession-data) that's updated everyweek (last updated 5 days ago) and an API (https://thesession.org/api) (seems to work, I quickly tested). The API page links to the database dump and some apps people have used the API for.

**Comment by liampond** [comment]
Added!
